src/hooks/useAuditoriumSeatedUsers.ts

```diff
@@ -0,0 +1,54 @@
+import { User } from "types/User";
+
+import { WithId } from "utils/id";
+import {
+  currentAuditoriumSectionSeatedUsersSelector,
+  emptyArray,
+} from "utils/selectors";
+
+import { useFirestoreConnect } from "hooks/useFirestoreConnect";
+import { useSelector } from "hooks/useSelector";
+
+export const findAuditoriumSeatedUsers = (
+  recentVenueUsers: readonly WithId<User>[],
+  venueId: string | undefined,
+  sectionId: string | undefined
+) => {
+  return sectionId && venueId
+    ? recentVenueUsers.filter(
+        (user) => user.data?.[venueId]?.sectionId === sectionId
+      )
+    : emptyArray;
+};
+
+const useConnectAuditoriumSectionSeatedUsers = (
+  venueId: string | undefined,
+  sectionId: string | undefined
+) => {
+  useFirestoreConnect(() => {
+    if (!venueId || !sectionId) return [];
+
+    return [
+      {
+        collection: "venues",
+        doc: venueId,
+        subcollections: [
+          { collection: "sections", doc: sectionId },
+          {
+            collection: "seatedUsers",
+          },
+        ],
+        storeAs: "currentAuditoriumSectionSeatedUsers",
+      },
+    ];
+  });
+};
+
+export const useAuditoriumSeatedUsers = (
+  venueId: string | undefined,
+  sectionId: string | undefined
```

Comment:
object

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/2318/commits/10fe3f04176fbe949d0025a18c5ea6761b3b44d8